docs/proposals/node-agent/README.md

```diff
@@ -0,0 +1,41 @@
+# Node Agent
+
+## Requirements
+
+### Which service must the domain provide
+
+The node agent amanges workload management. A workload is the tiniest unit in orka. It can be a container, a virtual machine, etc.
+
+### What is in scope and out of scope
+
+#### In scope
+
+The Node Agent provides an API :
+
+- to control the workloads :
+  - start
+  - stop
+  - restart
```

Comment:
You also want a `kill` operation on workloads.

Reply:
Agreed, fixed in : 412e498be7f48ac42ff799628d03add4272d6742